kairon/actions/definitions/voice_disconnect.py

```
@@ -0,0 +1,59 @@
+        self.bot = bot
+        self.name = name
+
+    def retrieve_config(self):
+        return {}
+
+    async def execute(self, dispatcher: CollectingDispatcher, tracker: Tracker, domain: Dict[Text, Any], **kwargs):
+        status = STATUSES.SUCCESS.value
+        exception = None
+        try:
+            dispatcher.utter_custom_json({"disconnect": True})
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)